gsa/src/gmp/models/__tests__/scanconfig.js

```diff
@@ -134,7 +135,7 @@ describe('ScanConfig model tests', () => {
     };
     const res = {
       count: 42,
-      trend: '1',
+      trend: SCANCONFIG_TREND_DYNAMIC,
```

Comment:
```suggestion
      trend: '1',
```

Reply:
That one was just fine :)

Tests are always `some plain input => expected output`. So if we get a string '1' from the backend we expect that our model converts this to `SCANCONFIG_TREND_DYNAMIC`